src/shared/components/VideoTileBase/VideoTileBase.tsx

```diff
@@ -203,196 +200,197 @@ export const VideoTileBase: React.FC<VideoTileBaseProps> = ({
-                      />
-                    ) : hasThumbnailUploadFailed ? (
-                      <CoverThumbnailUploadFailed>
-                        <SvgLargeUploadFailed />
-                        <Text variant="subtitle2" secondary>
-                          Thumbnail upload failed
-                        </Text>
-                      </CoverThumbnailUploadFailed>
-                    ) : (
-                      <CoverNoImage />
-                    )}
-                    {(videoPublishState === 'unlisted' || isDraft) && (
-                      <CoverVideoPublishingStateOverlay>
-                        {isDraft ? <SvgGlyphDraft /> : <SvgGlyphHide />}
-                        <PublishingStateText>{isDraft ? 'Draft' : 'Unlisted'}</PublishingStateText>
-                      </CoverVideoPublishingStateOverlay>
-                    )}
-                    {!!duration && <CoverDurationOverlay>{formatDurationShort(duration)}</CoverDurationOverlay>}
-                    <CoverHoverOverlay onClick={handleCoverHoverOverlayClick}>
-                      {publisherMode && (
-                        <CoverTopLeftContainer>
-                          <PullUp
-                            // set to true when video is already on the snackbar
-                            disabled={!!isPullupDisabled}
-                            onClick={(event) => {
-                              event.preventDefault()
-                              onPullupClick && onPullupClick(event)
-                            }}
-                          />
-                        </CoverTopLeftContainer>
-                      )}
-                      <CoverIconWrapper>
-                        {publisherMode ? (
-                          <SvgLargeEdit />
-                        ) : (
-                          <SvgOutlineVideo width={34} height={34} viewBox="0 0 34 34" />
-                        )}
-                      </CoverIconWrapper>
-                      {removeButton && (
-                        <RemoveButton onClick={handleRemoveClick}>
-                          <SvgGlyphClose />
-                        </RemoveButton>
-                      )}
-                    </CoverHoverOverlay>
-                  </Anchor>
-                </CoverImageContainer>
-              )}
-            </CSSTransition>
-          </SwitchTransition>
-        </CoverContainer>
-      </CoverWrapper>
-      {!!progress && (
-        <ProgressOverlay>
-          <ProgressBar style={{ width: `${progress}%` }} />
-        </ProgressOverlay>
-      )}
-      <SwitchTransition>
-        <CSSTransition
-          key={isLoading ? 'placeholder' : `content-${contentKey}`}
-          timeout={parseInt(transitions.timings.sharp)}
-          classNames={transitions.names.fade}
-        >
-          <InfoContainer main={main}>
-            {displayChannel && (
-              <AvatarContainer scalingFactor={scalingFactor}>
+    <div>
```

Comment:
What's this empty `div` for?

Reply:
In the end - I moved this div to `VideoGallery` because we still need this. The Carousel is messing with the width of slide parent and I still want to keep 100% or 78%(when it has `rankingNumber`) width on the VideoTile container:
![image](https://user-images.githubusercontent.com/51168865/129023079-07e579b5-1642-4770-b5ab-48ee051298bf.png)